src/components/region-common.js

```diff
@@ -1,74 +1,73 @@
+      [Utils.DEFAULT, () => ({ flag: unknownRegionFlag, regionDescription, computeZone: 'UNKNOWN', computeRegion: 'UNKNOWN' })]
+    )],
+    ['region', () => Utils.switchCase(location,
+      ['ASIA-EAST1', () => ({ flag: '🇹🇼', regionDescription: `${regionDescription} (Taiwan)`, computeZone: 'asia-east1-a', computeRegion: 'asia-east1' })],
+      ['ASIA-EAST2', () => ({ flag: '🇭🇰', regionDescription: `${regionDescription} (Hong Kong)`, computeZone: 'asia-east2-a', computeRegion: 'asia-east2' })],
+      ['ASIA-NORTHEAST1', () => ({ flag: '🇯🇵', regionDescription: `${regionDescription} (Tokyo)`, computeZone: 'asia-northeast1-a', computeRegion: 'asia-northeast1' })],
+      ['ASIA-NORTHEAST2', () => ({ flag: '🇯🇵', regionDescription: `${regionDescription} (Osaka)`, computeZone: 'asia-northeast2-a', computeRegion: 'asia-northeast2' })],
+      ['ASIA-NORTHEAST3', () => ({ flag: '🇰🇷', regionDescription: `${regionDescription} (Seoul)`, computeZone: 'asia-northeast3-a', computeRegion: 'asia-northeast3' })],
+      ['ASIA-SOUTH1', () => ({ flag: '🇮🇳', regionDescription: `${regionDescription} (Mumbai)`, computeZone: 'asia-south1-a', computeRegion: 'asia-south1' })],
+      ['ASIA-SOUTHEAST1', () => ({ flag: '🇸🇬', regionDescription: `${regionDescription} (Singapore)`, computeZone: 'asia-southeast1-a', computeRegion: 'asia-southeast1' })],
+      ['ASIA-SOUTHEAST2', () => ({ flag: '🇮🇩', regionDescription: `${regionDescription} (Jakarta)`, computeZone: 'asia-southeast2-a', computeRegion: 'asia-southeast2' })],
+      ['AUSTRALIA-SOUTHEAST1', () => ({ flag: '🇦🇺', regionDescription: `${regionDescription} (Sydney)`, computeZone: 'australia-southeast1-a', computeRegion: 'australia-southeast1' })],
+      ['EUROPE-NORTH1', () => ({ flag: '🇫🇮', regionDescription: `${regionDescription} (Finland)`, computeZone: 'europe-north1-a', computeRegion: 'europe-north1' })],
+      ['EUROPE-WEST1', () => ({ flag: '🇧🇪', regionDescription: `${regionDescription} (Belgium)`, computeZone: 'europe-west1-b', computeRegion: 'europe-west1' })],
+      ['EUROPE-WEST2', () => ({ flag: '🇬🇧', regionDescription: `${regionDescription} (London)`, computeZone: 'europe-west2-a', computeRegion: 'europe-west2' })],
+      ['EUROPE-WEST3', () => ({ flag: '🇩🇪', regionDescription: `${regionDescription} (Frankfurt)`, computeZone: 'europe-west3-a', computeRegion: 'europe-west3' })],
+      ['EUROPE-WEST4', () => ({ flag: '🇳🇱', regionDescription: `${regionDescription} (Netherlands)`, computeZone: 'europe-west4-a', computeRegion: 'europe-west4' })],
+      ['EUROPE-WEST6', () => ({ flag: '🇨🇭', regionDescription: `${regionDescription} (Zurich)`, computeZone: 'europe-west6-a', computeRegion: 'europe-west6' })],
+      ['NORTHAMERICA-NORTHEAST1', () => ({ flag: '🇨🇦', regionDescription: `${regionDescription} (Montreal)`, computeZone: 'northamerica-northeast1-a', computeRegion: 'northamerica-northeast1' })],
```

Comment:
Realize there is a NORTHAMERICA-NORTHEAST2 region now. Willy to review and update entire list.

Reply:
Can you try using NORTHAMERICA-NORTHEAST2 locally to create a runtime/PAPI jobs? I doubt it would work because Terra does not create VPC subnet in that region.

Reply:
Leo will create a VPC subnet once https://github.com/DataBiosphere/leonardo/pull/2319 is deployed. But it doesn't matter anyway, the UI will only allow one region per country to create runtimes. For Canada, we're only showing NORTHAMERICA-NORTHEAST1. See comment in https://github.com/DataBiosphere/terra-ui/pull/2642/files#diff-1fcbff2d49cbfcbbbd2e6fc3f78f453a36ae84030cbddfec06885366f9c00bc8R55